server/embed/config.go

```diff
@@ -762,6 +770,9 @@ func (cfg *Config) AddFlags(fs *flag.FlagSet) {
 	// unsafe
 	fs.BoolVar(&cfg.UnsafeNoFsync, "unsafe-no-fsync", false, "Disables fsync, unsafe, will cause data loss.")
 	fs.BoolVar(&cfg.ForceNewCluster, "force-new-cluster", false, "Force to create a new one member cluster.")
+
+	// featuregate
+	cfg.ServerFeatureGate.(featuregate.MutableFeatureGate).AddFlag(fs, ServerFeatureGateFlagName)
```

Comment:
If you follow https://github.com/etcd-io/etcd/pull/18279/files#r1670113740, then no need to do the type cast.
```suggestion
	cfg.ServerFeatureGate.AddFlag(fs, ServerFeatureGateFlagName)
```

Reply:
similar to https://github.com/etcd-io/etcd/pull/18279#discussion_r1670846400